addons/gdcef/gdcef/src/gdbrowser.cpp

```diff
@@ -176,6 +177,37 @@ void GDBrowserView::onPaint(CefRefPtr<CefBrowser> /*browser*/,
     m_texture->create_from_image(m_image, godot::Texture::FLAG_VIDEO_SURFACE);
 }
 
+
+//------------------------------------------------------------------------------
+  /// Called on the audio stream thread when a PCM packet is received for the
```

Comment:
Please use //! \brief instead of /// and move this inside the header file on the associated method